bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/ObserveTimeLineNode.java

```diff
@@ -93,13 +94,13 @@ public static long getDuration(PlannedTime pt, int step) {
 
             buf.append("<table><tr><th colspan=\"2\">Event</th><th colspan=\"2\">Sec</th></tr>");
             Step s = plannedTime.steps.get(step);
-            Map<Category, CategorizedTime> m = s.times.maxTimes()._1();
-            Map<Category, ImList<CategorizedTime>> losers = s.times.maxTimes()._2();
+            Map<Category, ImList<CategorizedTime>> m = s.times.groupTimes();
             for (Category c : Category.values()) {
-                CategorizedTime ct = m.get(c);
-                if (ct == null) continue;
+                ImList<CategorizedTime> cts = m.get(c);
+                if (cts == null) continue;
```

Comment:
Is there a better way to handle these nulls?

Reply:
I considered wrapping the `m.get` with `ImOption` but in the end I don't think it's worth exchanging the penalty in readability for the extra (little) safety that we get.